RFCs/use-prs-instead-of-issues.md

```diff
@@ -0,0 +1,8 @@
+# Use PRs instead of issues for RFCs
```

Comment:
> They're a whole new medium that we're not currently using, so there would be some cognitive overhead involved in introducing them (Having said that, maybe we should be using them more widely? Are there other needs we have that could be met with discussion? I think there's too much overlap with our Slack usage, and that discussions are best for wider communities rather than internal company discussions, but could be worthwhile investigating)

100% agree with this we don't need to put congitive overhead to anyone 👍 but worth investigating maybe at some point in the future? 🤔 

> You can't do inline, line-level comments - which is the main motivation for changing our approach in the first place

Well, we might be able to do it! Not with the way that we can do it on PRs but rather with the quote reply format ^ (pasting the line/code that we want to comment on) and the good thing with discussions is that we can start a thread from the comment like [this one](https://github.com/vercel/next.js/discussions/16832#discussioncomment-61499) that could act as an inline comment. 

Don't consider [this comment](https://github.com/artsy/README/pull/467#discussion_r899107249) blocking for this RFC, just wanted to put this into our radars because it looks super interesting!